bittensor/_receptor/receptor_pool_impl.py

```diff
@@ -133,28 +191,109 @@ def forward (
         # Init function.
         def call_forward( args ):
             return args['receptor'].forward( args['synapses'], args['inputs'], args['timeout'] )
+        
+        # Unpack responses
+        results_dict = dict(
+            outputs=[],
+            codes=[],
+            times=[],
+            uids=[]
+        )
```

Comment:
Using a dict here is a huuuge improvement to legibility.

Reply:
agreed